kubectl-cloudflow/cloudflowavro/avrocompatibility_test.go

```diff
@@ -0,0 +1,109 @@
+package cloudflowavro
+
+import (
+	"github.com/hamba/avro"
+	"github.com/stretchr/testify/assert"
+	"log"
+	"testing"
+)
+
+func Test_SimpleSchemaCompatibility(t *testing.T) {
+
+	readerSchema := `{
+	"type": "long",
+	"name": "data"
+}`
+
+	writerSchema := `{
+       "name": "data", 
+       "type": "int"
+}`
+
+	reader, err := avro.ParseWithCache(readerSchema, "", avro.DefaultSchemaCache)
+	writer, err := avro.ParseWithCache(writerSchema, "", avro.DefaultSchemaCache)
+	if err != nil {
```

Comment:
readerSchema could have had an err, line 23 overwrites err to nil, will not be detected.

Reply:
@RayRoestenburg this would happen if the two libraries are inconsistent. But you are right I will detect erros there.